### PR TITLE
Laravel 9.x Compatibility

### PR DESCRIPTION
**Pull request by laravel-shift**
This is an automated pull request from [Shift](https://laravelshift.com) to update your package code and dependencies to be compatible with Laravel 9.x.

**Before merging**, you need to:

- Checkout the `l9-compatibility` branch
- Review **all** comments for additional changes
- Thoroughly test your package

If you do find an issue, please report it by commenting on this PR to help improve future automation.

### CODE REVIEW

**Comment by laravel-shift** [comment]
:warning: Shift detected GitHub Actions which run jobs using a version `matrix`. Shift attempted to [update your configuration](https://github.com/tzsk/collage/pull/158/commits/17a24d548e455457a5bc32e26fddef5ff73ca758?w=1) for Laravel 9. However, you should review these changes to ensure the desired combination of versions are built for your package.

**Comment by laravel-shift** [comment]
:alembic: **Using this package?** If you would like to help test these changes or believe them to be compatible, you may update your project to reference this branch.

To do so, temporarily add Shift's fork to the `repositories` property of your `composer.json`:

```json
{
    "repositories": [
        {
            "type": "vcs",
            "url": "https://github.com/laravel-shift/collage.git"
        }
    ]
}
```

Then update your dependency constraint to reference this branch:

```json
{
    "require": {
        "tzsk/collage": "dev-l9-compatibility",
    }
}
```

Finally, run: `composer update`

File: .github/workflows/run-tests.yml [hunk 1]
```diff
@@ -9,10 +9,12 @@ jobs:
       fail-fast: true
       matrix:
         os: [ubuntu-latest]
-        php: [7.4, 8.0]
-        laravel: [7.*, 8.*]
+        php: [7.4, 8.0, 8.0]
```
Comment (jasonmccreary) [inline .github/workflows/run-tests.yml:12]: I'm not sure why it duplicates the PHP version here, but you can remove it. We'll get this fixed up for next time. 👍

Comment (tzsk) [reply .github/workflows/run-tests.yml:12]: Yeah. Sure. Thanks 👍

**Pull request merged**